src/main/webapp/app/exercises/quiz/manage/quiz-exercise-detail.component.ts

```diff
@@ -783,6 +813,25 @@ export class QuizExerciseDetailComponent implements OnInit, OnChanges, Component
                 translateValues: {},
             });
         }
+        const maxPointsReachableInQuiz = this.quizExercise.quizQuestions
+            ?.map((quizQuestion) => {
+                if (quizQuestion.score === undefined || quizQuestion.score === null) {
```

Comment:
Why not just `return quizQuestion.score ?? 0` ?

Reply:
good catch!